### PR TITLE
Removed host networking

### PR DESCRIPTION
**Pull request by MuratUrsavas**
**Checklist**

- [ ] Tests added
- [ ] Cleaned up commit history (rebase!)
- [ ] Documentation added
- [ ] Thought about variable and method names

### CODE REVIEW

File: templates/docker-compose.template [hunk 1]
```diff
@@ -59,20 +56,21 @@ services:
     environment:
       - FIRMWARE_VERSION={{FIRMWARE_VERSION}}
       - FIRMWARE_SHORT_HASH={{ENV.FIRMWARE_SHORT_HASH}}
+    ports:
+      - "1681:1681"
```
Comment (shawaj) [inline templates/docker-compose.template:60]: we should leave as expose not ports...as we dont want to expose externally

File: templates/docker-compose.template [hunk 2]
```diff
@@ -59,20 +56,21 @@ services:
...
 
   helium-miner:
     image: nebraltd/hm-gatewayrs:{{ARCH}}-{{GATEWAYRS_VERSION}}
     depends_on:
       - diagnostics
-    expose:
-      - "1680"
-      - "4467"
```
Comment (shawaj) [inline templates/docker-compose.template:69]: same as above, these should stay the same

File: templates/docker-compose.template [hunk 3]
```diff
@@ -95,8 +93,6 @@ services:
       - miner-storage:/var/data
       - watchdog-log:/var/watchdog
       - nebra:/var/nebra
-    ports:
```
Comment (shawaj) [inline templates/docker-compose.template:98]: why move to end?

**Review by shawaj — CHANGES_REQUESTED** [review]

File: templates/docker-compose.template (hunk 1 above)
Comment (shawaj) [reply templates/docker-compose.template:60]: AFAIK expose exposes to other containers but ports exposes them external to the device - which we dont want

File: settings.ini [hunk 4]
```diff
@@ -1,7 +1,7 @@
 [versions]
-FIRMWARE_VERSION=v1.0.2
+FIRMWARE_VERSION=v1.0.2-1
 DIAGNOSTICS_VERSION=0739ab9
-CONFIG_VERSION=6d63852
+CONFIG_VERSION=5d3ad7a
```
Comment (shawaj) [inline settings.ini:4]: it is weird that this is showing as a diff....when this is already how it is in master

File: templates/docker-compose.template (hunk 1 above)
Comment (MuratUrsavas) [reply templates/docker-compose.template:60]: I've checked. expose is now used decoratively. The functionality is the same as forwarding. That's why I've used it under ports.

Comment (MuratUrsavas) [reply templates/docker-compose.template]: Ok done.

File: templates/docker-compose.template (hunk 3 above)
Comment (MuratUrsavas) [reply templates/docker-compose.template]: No specific reason :smile: Just appended. Now noved up.

**Review by shawaj — APPROVED** [review]

**Comment by shawaj** [comment]
LGTM

File: settings.ini (hunk 4 above)
Comment (MuratUrsavas) [reply settings.ini:4]: Yeah. The base is one commit before but it should be diffing to HEAD. It gets confused today I guess, like we all did :)

Comment (shawaj) [reply settings.ini:4]: doesn't really matter - just weird...will dismiss

**Pull request merged**